lib/Model.js

```diff
@@ -105,6 +112,7 @@ class Model {
           if (!advisoryMap.has(via.source)) {
             advisoryMap.set(via.source, {
               id: via.source,
+              github_advisory_id: via.url.split("/")[4],
```

Comment:
Adding the `github_advisory_id` based on the URL here.

Reply:
Perhaps we can move this `url.split("/")[4]` into a dedicated function like:

```ts
function convertGitHubAdvisoryUrlToAdvisoryId(url) {
  return url.split("/")[4];
}
```

Reply:
Done